tools/run_tests/run_xds_tests.py

```diff
@@ -2304,6 +2305,39 @@ def test_csds(gcp, original_backend_service, instance_group, server_uri):
                             ok = False
                         else:
                             seen.add('eds')
+                for generic_xds_config in client_config['GenericXdsConfig']:
+                    if re.search(r'\.Listener$', generic_xds_config['typeUrl']):
+                        seen.add('lds')
+                        listener = generic_xds_config["xds_config"]
+                        if listener['name'] != server_uri:
+                            logger.info('Invalid Listener name %s != %s',
+                                        listener_name, server_uri)
+                            ok = False
+                    elif re.search(r'\.RouteConfiguration$',
+                                   generic_xds_config['typeUrl']):
+                        seen.add('rds')
+                        route_config = generic_xds_config["xds_config"]
+                        if len(route_config['virtual_hosts']) <= 0:
```

Comment:
Nit: How could the length be less than 0?

Reply:
Changed to `if not len(route_config['virtual_hosts']):`.

I was trying to be thorough, but this line is unnecessary unless CPython broke.